Configurations/platform/Unix.pm

```diff
@@ -63,7 +63,9 @@ sub sharedname_simple {
 }
 
 sub sharedlib_simple {
-    return undef if $_[0]->shlibext() eq $_[0]->shlibextsimple();
+    return undef
+        if ($_[0]->sharedname($_[1]) eq $_[0]->sharedname_simple($_[1])
```

Comment:
Ah, sharedname_simple can return `undef` so that should be handled as well.

Reply:
Ah, good point.  Fixed